lib/PkgConfig.pm

```diff
@@ -270,12 +270,21 @@ if($ENV{PKG_CONFIG_NO_OS_CUSTOMIZATION}) {
     require Config;
     if($Config::Config{myuname} =~ /strawberry-perl/)
     {
-        my($vol, $dir, $file) = File::Spec->splitpath($^X);
-        my @dirs = File::Spec->splitdir($dir);
-        splice @dirs, -3;
-        my $path = (File::Spec->catdir($vol, @dirs, qw( c lib pkgconfig )));
-        $path =~ s{\\}{/}g;
-        @DEFAULT_SEARCH_PATH = $path;
+        #  handle PAR::Packer executables which have $^X eq "perl.exe"
+        if ($ENV{PAR_0})
```

Comment:
as mentioned, `$ENV{PAR_0}` is only handled with Strawberry.  I think this is the correct thing to do.  Strawberry Perl is a strange case where we treat Strawberry as the "operating system" in a sense, since it provides `.pc` files, while the the actual operating system does not (typically). :+1: